site/src/layouts/types/index.ts

```diff
@@ -13,9 +13,7 @@ export type LayoutProps = {
   ToCProps?: {
     items: any;
   };
```

Comment:
@DavieReid Can we get this type exported from Mosaic?

Reply:
yeah we can export LayoutProps